client/app/initialize.coffee

```diff
@@ -11,27 +11,44 @@ Logging
 
 Send client side errors to server.
 ###
-window.onerror = (msg, url, line, col, error) ->
-    console.error msg, url, line, col, error, error?.stack
-    exception = error?.toString() or msg
-    if exception isnt window.lastError
+initReporting = ->
+    env     = require('imports').env
+    console = console or window.console or {}
+
+    proto  = console
```

Comment:
Same response as below. fact is that the console doesn't really expose a prototype, so we directly override its methods, we can skip this step too.